src/content/docs/en/guides/content-collections.mdx

```diff
@@ -208,11 +212,12 @@ Schemas also power Astro's automatic TypeScript typings for your content. When y
 In order for Astro to recognize a new or updated schema, you may need to restart the dev server or [sync the content layer](/en/reference/cli-reference/#astro-dev) (<code>s + enter</code>) to define the `astro:content` module.
 :::
 
-Every frontmatter or data property of your collection entries must be defined using a Zod data type:
+Every frontmatter or data property of your collection entries must be defined using a [Zod data type](/en/reference/modules/astro-zod/#common-data-type-validators):
 
-```ts title="src/content.config.ts" {6-11,15-19}
-import { defineCollection, z } from 'astro:content';
+```ts title="src/content.config.ts" {7-12,16-20}
+import { defineCollection } from 'astro:content';
 import { glob, file } from 'astro/loaders'; // Not available with legacy API
```

Comment:
```suggestion
import { glob, file } from 'astro/loaders';
```
This can probably go, even now? (For sure we should make sure this is not in v6 (checked and yes, it's gone!), but I think it's safe to remove now.)